python/paddle/nn/functional/__init__.py

```diff
@@ -97,6 +97,7 @@
 # from .extension import temporal_shift  #DEFINE_ALIAS
 # from .extension import warpctc  #DEFINE_ALIAS
 from .extension import diag_embed  #DEFINE_ALIAS
+from .extension import sequence_mask  #DEFINE_ALIASs
```

Comment:
`#DEFINE_ALIAS` is no longer needed, also, a typo (`s`)

Reply:
Done